cpp/test/phonenumbers/phonenumberutil_test.cc

```diff
@@ -66,7 +66,12 @@ class PhoneNumberUtilTest : public testing::Test {
 
   const PhoneMetadata* GetMetadataForNonGeographicalRegion(
       int country_code) const {
-    return phone_util_.GetMetadataForNonGeographicalRegion(country_code);
+    return phone_util_.GetMetadataForNonGeographicalEntity(country_code);
+  }
```

Comment:
You can also remove this because you already renamed the calls to this function.